### ISSUE TITLE
Compatibility with other mods adding logs (e.g. Regions Unexplored or Terrestria)

### ISSUE THREAD

**Issue by 7sina** [Globox1997/EarlyStage#13]
### Make sure you are not opening a duplicate.

- [X] I have searched the issue tracker and did not find an issue describing my bug.

### Description of your suggestion.

Hello, as suggested above there is no compatibility with mods adding logs such as Regions Unexplored, Better End/Nether or Terrestria which makes the mod totally useless since you can just get wood like vanilla with those added by these mods..

**Comment by polykatt** [comment]
I agree this issue does break the mod almost entirely. Hope this will be fixed soon

**Comment by Tar1q22** [comment]
Thirding this, forgive my ignorance but isn't it as simple as adding the correct tag? I had a look at the code to see if I could do it myself but I don't know where the no tree punching is.

**Comment by Globox1997** [comment]
Noticed.

**Comment by Globox1997** [comment]
Not sure about regions unexplored but terrestria is compatible on version 1.21.1

**Comment by theDerxu** [comment]
would it be possible to just implement using the `#logs` tag in some way? I'd messed around with datapacks but nothing seemed to fully work with everything, but most mods should have all their logs under that tag so if there was a way to do that.. but im not an expert

**Comment by Globox1997** [comment]
Added compat with [https://github.com/Globox1997/EarlyStage/commit/77949d554a42dd85753afc28026285ba681e7433](https://github.com/Globox1997/EarlyStage/commit/77949d554a42dd85753afc28026285ba681e7433)
